x-pack/plugins/fleet/server/services/epm/packages/get.ts

```diff
@@ -126,6 +126,7 @@ export async function getPackageInfo(options: {
     title: packageInfo.title || nameAsTitle(packageInfo.name),
     assets: Registry.groupPathsByService(paths || []),
     removable: !isRequiredPackage(pkgName),
+    id: packageInfo.name,
```

Comment:
This mimics the behavior in the list endpoint: https://github.com/elastic/kibana/blob/master/x-pack/plugins/fleet/server/services/epm/packages/get.ts#L51